android/src/main/java/com/stripeterminalreactnative/listener/RNOfflineListener.kt

```diff
@@ -25,6 +26,10 @@ class RNOfflineListener(
     override fun onPaymentIntentForwarded(paymentIntent: PaymentIntent, e: TerminalException?) {
         context.sendEvent(ReactNativeConstants.FORWARD_PAYMENT_INTENT.listenerName) {
             putMap("result", mapFromPaymentIntent(paymentIntent, ""))
+            putMap("error", nativeMapOf {
+                putString("code", e?.errorCode.toString())
+                putString("message", "hahahaha")
```

Comment:
This appears to be a hard-coded string for debugging. 😅

Reply:
awkward😅